prov/gni/test/rdm_atomic.c

```diff
@@ -481,7 +484,7 @@ void do_min(int len)
 
 	cr_assert_eq(ret, 1, "fi_cq_read returned %d %d", ret,
 			dump_cq_error(send_cq[0], target, 0));
-	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0);
+	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0, ep[1]);
```

Comment:
Ok. Do we want to change these tests to report completions on the remote peer and read those completions or simply pass ep[0] into this test and others like it?